Tools/MobileMiniBrowser/MobileMiniBrowserFramework/WebViewController.m

```diff
@@ -222,6 +224,36 @@ - (void)removeWebView:(WKWebView *)webView
     [self.tabViewController.tableView reloadData];
 }
 
+- (NSString *)addProtocolIfNecessary:(NSString *)address
```

Comment:
It's a shame this implementation can't be shared with normal MiniBrowser.

Reply:
Yeah I copy/pasted it directly from Mac MiniBrowser. Maybe the SwiftUI rewrite can unify the code for mac and iOS.